docs/static/hour-of-code/2021/styles.css

```diff
@@ -683,19 +806,27 @@ a.activity-btn .title {
 }
 
 a.activity-btn .body {
-    height: 30rem;
+    height: 33rem;
     margin-top: -3rem;
-    border-top-left-radius: 2rem;
-    border-top-right-radius: 2rem;
+    border-radius: 2rem;
+    background-image: url('./forest-skillmap-game.gif');
```

Comment:
the css file doesn't get these replaced so it won't point at the right thing; it'll probably be much easier to move these css rules directly into the html files while iterating as otherwise they will always be slightly out of sync due to caching (and then it will go through the same link patching logic as the rest of the html)

Reply:
Ah sure thing, I'll move to HTML